aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -113,6 +115,17 @@ impl BlockExecutorTransactionOutput for AptosTransactionOutput {
             .collect()
     }
 
+    /// Should never be called after incorporate_delta_writes, as it
```

Comment:
The comments for this and the above get functions seem wrong / outdated. Doesn't seem like we consume the output at all.

Reply:
oh nevermind, it's just bad phrasing, meaning that incorporate_delta_writes is the one that consumes. Intended to say like for fee_statement, that it can't be used after vm_output is consumed.